public/app/plugins/datasource/influxdb/components/FluxQueryEditor.tsx

```diff
@@ -60,7 +60,7 @@ v1.measurements(bucket: v.bucket)`,
     label: 'Schema Exploration: (fields)',
     description: 'Return every possible key in a single table',
     value: `from(bucket: v.bucket)
-  |> range(start: v.timeRangeStart, stop:timeRangeStop)
+  |> range(start: v.timeRangeStart, stop:v.timeRangeStop)
```

Comment:
this missing `v.` is a typo in our suggested queries. 

I added the change here just to reduce the size of #26634